src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -172,5 +213,82 @@ public string WorkingDirectory
             get { return _directory ?? string.Empty; }
             set { _directory = value; }
         }
+
+        public bool ErrorDialog
+        {
+            get { return _errorDialog; }
+            set { _errorDialog = value; }
+        }
+
+        public IntPtr ErrorDialogParentHandle 
+        {
+            get { return _errorDialogParentHandle; }
+            set { _errorDialogParentHandle = value; }
+        }
+
+        [DefaultValueAttribute("")]
+        public string Verb 
+        {
+            get 
+            {
+                if (_verb == null) return string.Empty;
+                return _verb;
+            }
+            set 
+            {
+                _verb = value;
+            }
+        }
+
+        public string[] Verbs {
```

Comment:
This registry-based implementation is Windows-specific and should be moved into ProcessStartInfo.Windows.cs.  Then a member with the same signature should be added to the .Unix.cs file, and it can just do:
```C#
public string[] Verbs => Array.Empty<string>();
```